src/migrations/neo/movies.js

```diff
@@ -0,0 +1,91 @@
+const request = require('request');
+const rp = require('request-promise');
+const {BASE_URL} = require('../../config');
+
+async function deleteNeoModels(type) {
+  return rp.del({
+    url: BASE_URL + `/neo/${type}`,
+    json: true
+  });
+}
+
+async function insertNeoModel(type, model) {
+  return rp.post({
+    url: BASE_URL + `/neo/${type}`,
+    form: model,
+    json: true
+  })
+  console.log('inserted model')
+}
+
+async function insertNeoRole(actor, movie, roles) {
+  let urlEnd = roles == null ? '' :  `?roles="${encodeURIComponent(roles)}"`;
+  return rp.post({
+    url: BASE_URL + `/neo/actors/${actor}/movies/${movie}` + urlEnd,
```

Comment:
Does the `/neo` in the URL refer to same thing as the db in the `fetchMovies` function?